.gitlab-ci.yml

```diff
@@ -180,7 +180,7 @@ test_rust:
     - RUST_BACKTRACE=1 cargo test
     - cd $CI_PROJECT_DIR
     - git clone https://github.com/Blockstream/electrs electrs-elements
-    - cd electrs-elements && git checkout 5bae341585f70699cf12b587a1e9d392df43d674
+    - cd electrs-elements && git checkout d6043203310168bee18e47c19772ef375b9daaee
```

Comment:
Should we upgrade to https://github.com/Blockstream/electrs/commit/1cbc86fbfdb5680fea752377bf8f18ed417f0edd?

Reply:
Updated to the latest (https://github.com/Blockstream/electrs/commit/65e20f9e1196c0039d56307b34ea7bcfbe375b99) in 70c9c23fe1934628672d05cb4fb41e71d085b8af.